src/components/Badge/FreeBookRenderer.tsx

```diff
@@ -25,13 +25,12 @@ const freeBookStyle = css`
   line-height: 12px;
 `;
 
-// 무료책 권수 썸네일에 표시하는 컴포넌트
 const FreeBookRenderer: React.FC<FreeBookRendererProps> = (props) => {
   if (props.freeBookCount && props.freeBookCount > 0) {
     return (
-      <span css={freeBookStyle} aria-label={`${props.freeBookCount}${props.unit} 무료`}>
+      <FreeBookBadge aria-label={`${props.freeBookCount}${props.unit} 무료`}>
```

Comment:
```suggestion
      <FreeBookBadge>
```

텍스트를 그대로 읽으면 되므로 따로 레이블을 지정할 필요가 없습니다. 그리고 `span` 엘리먼트는 `role="img"`가 아니면 `aria-label`이 작동하지 않아요.

Reply:
아하 그랬군요! 몰랐습니다! 🙏

Reply:
> They are OK on `div` elements **IF** they have `role=navigation`, `search`, `main`, `img`.

`span`도 안될수도있겠네요

Reply:
아 `span`은 되는것같아요